docs/manuals/source/Appendix/Howtos/RemovingAClient.rst.inc

```diff
@@ -0,0 +1,37 @@
+.. _section-RemoveClient:
```

Comment:
Shouldn't we use the same name for the file RemoveClient.rst.inc (the use of AClient doesn't seems to bring any useful information, even can add confusion in reading)

Reply:
Surely RemovingAClient looks ugly, however we specified to name the File in CamelCase after the first heading, see https://docs.bareos.org/DocumentationStyleGuide/BareosSpecificFormatting/DocumentFileStructure.html